indra/assemblers/pybel_assembler.py

```diff
@@ -171,6 +190,9 @@ def _assemble_active_form(self, stmt):
     def _assemble_complex(self, stmt):
         """Example: complex(p(HGNC:MAPK14), p(HGNC:TAB1))"""
         complex_data, _ = _get_complex_node(stmt.members)
+        if complex_data is None:
+            logger.info('skip adding complex with no members: %s', stmt.members)
+            return
         self.model.add_node_from_data(complex_data)
```

Comment:
Interesting--do you have any examples of the complexes where this has occurred? I assume this is because of the failure to generate one or more nodes.

Reply:
These all had problems:

```
Activation(CD106(bound: [CD29, True]), adhesion())
Inhibition(GAPs__GTPase_activating_proteins(bound: [Rac1_GTP, True]), RAC1())
Activation(rhHSP60(bound: [OPCs, True]), apoptosis())
Activation(XBP1s(bound: [its, True]), transcription())
```

Reply:
Aha! This must be because we haven't yet implemented grounding mapping or filtering for ungrounded entities in BoundConditions--see issue #149.

Reply:
Thanks for looking into that. Where do you think the best place to check if statements like these should be skipped? For some reason, placing this if statement here doesn't sit well with me since it doesn't really document what the problem is

Reply:
If the issue is what I think it is, you would have seen the warning "Agent foo has no grounding" from _get_agent_node_no_bcs, below (line 312). Did that show up in the logs?

Reply:
Indeed. Here's a link to the pickle of the offending IncrementalModel I made with the RAS machine: https://owncloud.scai.fraunhofer.de/index.php/s/5jJHSwIQ2Ci1Uf5